images/win/scripts/Installers/Validate-Haskell.ps1

```diff
@@ -0,0 +1,52 @@
+################################################################################
+##  File:  Validate-Haskell.ps1
+##  Desc:  Validate Haskell for Windows
+################################################################################
+
+# GHC validation
+if ((Get-Command -Name 'ghc'))
+{
+    Write-Host "ghc is on the path"
+}
+else
+{
+    Write-Host "ghc is not on path."
+    exit 1
+}
+
+$SoftwareName = "ghc"
+[String] $DefaultGhcVersion = & ghc --version
+$ChocoPackagesPath = Join-Path $env:ChocolateyInstall "lib"
+$GhcVersionList = Get-ChildItem -Path $ChocoPackagesPath -Filter "ghc.*" | ForEach-Object { $_.Name.TrimStart("ghc.") }
```

Comment:
I think it makes sense to add additional validation to check that accurate 3 versions are installed.